Gems/Atom/Tools/AtomToolsFramework/Code/Source/PreviewRenderer/PreviewRendererCaptureState.cpp

```diff
@@ -20,7 +20,11 @@ namespace AtomToolsFramework
 
     PreviewRendererCaptureState::~PreviewRendererCaptureState()
     {
-        AZ::Render::FrameCaptureNotificationBus::Handler::BusDisconnect();
+        if (m_frameCaptureId != AZ::Render::InvalidFrameCaptureId)
+        {
+            AZ::Render::FrameCaptureNotificationBus::MultiHandler::BusDisconnect(m_frameCaptureId);
+            m_frameCaptureId = AZ::Render::InvalidFrameCaptureId;
+        }
```

Comment:
This is only doing one at a time so you don't need a multi handler. You also don't need to check if the ID is valid. You can just call disconnect without passing in the ID. No need to reset the frame capture ID either. You really only need it to connect the notification bus.